### PR TITLE
Fix bug: resgroup total wait during always 0.

### PR DESCRIPTION
**Pull request by kainwen**
DEV CI https://dev.ci.gpdb.pivotal.io/teams/main/pipelines/zlv_test_rg_wait_time

### CODE REVIEW

File: src/test/isolation2/sql/resgroup/resgroup_wait_time.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+3:end;
+1q:
+2q:
+3q:
+
+drop role role_wait_time_test;
+drop resource group rg_wait_time_test;
+
+create resource group rg_wait_time_test with (concurrency=2, cpu_max_percent=20);
```
Comment (SmartKeyerror) [inline src/test/isolation2/sql/resgroup/resgroup_wait_time.sql:33]: What's the difference in this case between the above one? It seems duplicated.

Comment (kainwen) [reply src/test/isolation2/sql/resgroup/resgroup_wait_time.sql:33]: The first is testing wait-and-then-get-slot, the second is testing wait-and-canceld.

**Review by SmartKeyerror — APPROVED** [review]
LGTM.

**Comment by kainwen** [comment]
DEV CI passed, push this. Thanks for review!

**Pull request merged**